src/wallet/scriptpubkeyman.h

```diff
@@ -643,6 +643,7 @@ class DescriptorScriptPubKeyMan : public ScriptPubKeyMan
 
     const WalletDescriptor GetWalletDescriptor() const EXCLUSIVE_LOCKS_REQUIRED(cs_desc_man);
     const std::unordered_set<CScript, SaltedSipHasher> GetScriptPubKeys() const override;
+    const std::unordered_set<CScript, SaltedSipHasher> GetScriptPubKeys(int32_t minimum_index) const;
```

Comment:
```suggestion
    const std::unordered_set<CScript, SaltedSipHasher> GetScriptPubKeys(int32_t minimum_index = 0) const;
```

Reply:
In that case, it would be also need to change `LegacyScriptPubKeyMan::GetScriptPubKeys` to `LegacyScriptPubKeyMan::GetScriptPubKeys(int32_t minimum_index)`

Reply:
> In that case, it would be also need to change `LegacyScriptPubKeyMan::GetScriptPubKeys` to `LegacyScriptPubKeyMan::GetScriptPubKeys(int32_t minimum_index)`

Yes, we would need to impose the parameter on the base class, which I think wouldn't make much sense (the `minimum_index` for this method is only useful for descriptor wallets). Also see the discussion here: https://github.com/bitcoin/bitcoin/pull/25957#discussion_r964582588